app.js

```diff
@@ -79,9 +80,9 @@ function fetchInfo(url) {
 //Helper Function ---------------------------------------------------------------------------------------------------------------------
 
 function generateList(data) {
-    let html2 = '';
+    let html2 = ''; // Comment on naming
     data.filter(element => element.show.image !== null) //filter out shows that have no image to display
-        .forEach(element => {
+        .forEach(element => {  // Comment on map
```

Comment:
Instead of using `forEach` here, you could also use the `reduce` function like this:

```
let html2 = data.filter(element => element.show.image !== null)
        .reduce((finalString, currentShow) => {  // Comment on map
          finalString += // all that HTML for displaying a show
        })
```

[Documentation about reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)